src/katsdpcontroller/fake_servers.py

```diff
@@ -108,6 +117,16 @@ def __init__(self, *args, **kwargs) -> None:
                 )
             )
 
+        self.sensors.add(
+            Sensor(
+                DeviceStatus,
+                "rx.device-status",
+                "The F-engine is receiving a good, clean digitiser stream",
+                default=DeviceStatus.DEGRADED,
+                initial_status=Sensor.Status.WARN,
+            )
```

Comment:
I was having a general thought about "why these values should be what they are", and I see that the ICD doesn't actually mention default value/status combinations for sensors. Do we know why that is?

Reply:
Do you mean why the ICD doesn't mention default values? Well, what the sensors show should reflect what is actually going on. This is just a "fake" server so it's got to have some values to work with.

In this case, the device-status sensor is supposed to be a function of what is underneath it, and since there are some sensors that are in `ERROR`, I made the status of this correspond to what it would in a real case.

Reply:
> Do you mean why the ICD doesn't mention default values?

Indeed indeed. Although, looking at fgpu's [populate_sensors](https://github.com/ska-sa/katgpucbf/blob/main/src/katgpucbf/fgpu/engine.py#L723-L786), I see the foundation of my point crumbling as
* there doesn't seem to be any consistency on whether a sensor needs any combination of `default` and `initial_status` (upon instantiation).

Not to say that it *should* require either/both of those.

> In this case, the device-status sensor is supposed to be a function of what is underneath it, and since there are some sensors that are in `ERROR`, I made the status of this correspond to what it would in a real case.

Indeed indeed once again. I follow your logic (and also checked why these values were so during my review) - no qualms there.